### PR TITLE
Render basic graphical objects

### LINKED ISSUE #201: Render basic graphical objects
#### Description:
Support rendering of basic graphical objects described in https://github.com/BHoM/BHoM/issues/1190.

### PR DESCRIPTION
**Pull request by rolyhudson**
### NOTE: Depends on 
https://github.com/BHoM/BHoM/pull/1191
To view these objects in Rhino / Grasshopper
https://github.com/BHoM/Grasshopper_Toolkit/pull/602

   
### Issues addressed by this PR

Closes #201

### CODE REVIEW

File: Rhinoceros_Engine/Convert/ToRhino.cs [hunk 1]
```diff
@@ -610,6 +620,46 @@ public static List<object> ToRhino(this BHG.CompositeGeometry geometries)
             return geometries.Elements.Select(x => x.IToRhino()).ToList();
         }
 
+        /***************************************************/
+        /**** Representation  fallback                  ****/
+        /***************************************************/
+
+        public static object ToRhino(this IRender render)
```
Comment (alelom) [inline Rhinoceros_Engine/Convert/ToRhino.cs:627]: This should be private, I think. We should not expose fallback in the UI.

Comment (alelom) [reply Rhinoceros_Engine/Convert/ToRhino.cs:627]: If you agree with the above, this method should be moved among the private methods section below the public ones. The header should read "Private methods - fallback"

Comment (alelom) [reply Rhinoceros_Engine/Convert/ToRhino.cs:627]: Having had a better look, I can't seem to find any fallback for `ToRhino(IGeometry ...)`!

I think our approach could be to add a single fallback that accepts `IObject` so we cover both `IGeometry` and `IRender`.

File: Rhinoceros_Engine/Convert/ToRhino.cs [hunk 2]
```diff
@@ -610,6 +620,46 @@ public static List<object> ToRhino(this BHG.CompositeGeometry geometries)
...
+        {
+            return null;
+        }
+
+        /***************************************************/
+        public static Text3d ToRhino(this RenderText renderText)
```
Comment (alelom) [inline Rhinoceros_Engine/Convert/ToRhino.cs:633]: Could you group these public method under a header that reads "Public methods - IRender" to follow the convention for the methods in this file?

**Review by alelom — CHANGES_REQUESTED** [review]
Just a few comments

**Review by alelom — APPROVED** [review]
Perfect. Thanks!

**Comment by rolyhudson** [comment]
@BHoMBot check all

**Comment by rolyhudson** [comment]
@BHoMBot check-core

**Comment by rolyhudson** [comment]
@BHoMBot check core

**Pull request merged**